taxi_service/settings.py

```diff
@@ -122,7 +122,11 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
+
+STATICFILES_DIRS = [
+    BASE_DIR / "static",
+]
```

Comment:
The `STATICFILES_DIRS` setting is correctly pointing to the 'static' directory. This ensures that Django can find your static files during development.